test/CallbackStreamTest.php

```diff
@@ -89,13 +89,13 @@ public function testGetSize()
         $this->assertNull($ret);
     }
 
-    /**
-     * @expectedException RuntimeException
```

Comment:
I'll allow it. Semantically, having the call to `expectException()` immediately prior to the code that raises the exception is better. While I realize it's unrelated, since this is updating the PHPUnit version already, we might as well do some cleanup.